index.html

```diff
@@ -241,7 +241,7 @@ <h3 class="frameworks">Skills</h3>
         </div>
     </section>
 
-    <section class="contact-form">
+    <section class="contact-form" id="Contact">
         <p class="footer-text">
             I'm always interested in hearing about new projects, so if you'd like to
             chat please get in touch.
```

Comment:
- [ ] Great effort working on your contact form project, but currently I could not submit some data to the form, Kindly check the name attribute you have given the input form and be sure they are correct, Also the styling still requires some attention in order for it to look like what we have in the original design. Kindly work on it and be sure to have a perfect project before resubmitting it for review.

 Below is a video explanation 👇 

[screen-capture (1).webm](https://github.com/okemboi/portfolio/assets/78634841/5530f9b4-8afc-4ac2-b5e9-434708010c6b)